crates/cxx-qt-lib/src/core/qt.rs

```diff
@@ -22,6 +22,25 @@ mod ffi {
         CaseSensitive,
     }
 
+    /// This enum describes the types of connection that can be used with signals.
+    ///
+    /// Note that UniqueConnection is not supported as
```

Comment:
```suggestion
    /// Note that UniqueConnection is not supported as we only support Lambda functors at the moment.
```

Reply:
haha wonder what happens there must have not flushed the file :-)